content/docs/api-reference/c-chain/api.mdx

```diff
@@ -81,6 +81,14 @@ For batched requests on the [public API node](/tooling/rpc-providers) , the maxi
 number of items is 40. We are working on to support a larger batch size.
 </Callout>
 
+#### Exceptions
+
+- `eth_getProof` behaves differently than geth, from release v0.14.1, with the following differences:
+  - On archive nodes (`"pruning-enabled": false`), queries for proofs for state older than approximately 24 hours preceeding the last accepted block will be rejected by default.
+    This can be adjusted with the option `historical-proof-query-window` which defines the number of blocks before the last accepted block which should be accepted for state proof queries, or set it to `0` to accept any block number state query.
+  - On pruning nodes (`"pruning-enabled": true`), queries for proofs past the tip buffer of 32 blocks are always rejected
+
+
```

Comment:
```suggestion
`eth_getProof` behaves differently than geth, from release `v0.14.1`, with the following differences:
- On archival nodes (nodes with`pruning-enabled` set to `false`), queries for state proofs older than 24 hours preceding the last accepted block will be rejected by default. This can be adjusted with `historical-proof-query-window`, which defines the number of blocks before the last accepted block that can be queried for state proofs. Set this option to `0` to accept a state query for any block number.  
- On pruned nodes (nodes with `pruning-enabled` set to `true`), queries for proofs past a 32 block buffer are always rejected.

```

Reply:
1. TIL I've been using `preceeding` wrong forever, thanks! Only 1 `e`!
1. Isn't it `pruning nodes` instead of `pruned nodes`? We're talking about avalanche nodes, not i.e. trie nodes 🤔 
1. `queries for state proofs past a 32 block buffer are always rejected.` ➡️  `queries for state proofs outside the 32 blocks window trailing the last accepted block are always rejected.` - since the `32 block buffer` might leave some room for interpretation and is a bit of an implementation details I'd think?

Thanks!

Reply:
1. 🫡 
2. 🫡 
3. corrected
4. reworded a bit differently

Reply:
please "sign off and commit merge" if the changes are good to go